src/components/ContentHighlights/CurrentContentHighlightItemsHeader.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Button } from '@edx/paragon';
+import { useParams } from 'react-router-dom';
+import { TEST_COURSE_HIGHLIHTS_DATA } from './data/constants';
+
+const CurrentContentHighlightItemsHeader = () => {
+  const { highlightUUID } = useParams();
+  const [highlightTitle] = useState(TEST_COURSE_HIGHLIHTS_DATA.filter(
+    highlights => highlights.uuid === highlightUUID,
+  )[0].title);
+  return (
+    <>
+      { highlightTitle && (
+        <header>
+          <div className="d-flex justify-content-between mb-2">
```

Comment:
[curious] similar to the other header component, could we use `ActionRow` here? feels like an applicable use case 🤔